### PR TITLE
Papa 6 ingest csvs

### PR DESCRIPTION
**Pull request by cdobs**
Papa 6 ingest csvs were missing from this repo. The data have already been ingested. If any mistakes are found, purge and re-ingest will be required.

### CODE REVIEW

File: GP02HYPM/GP02HYPM_R00006_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/C*.DA?,GP02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/A*.DA?,GP02HYPM-WFP03-05-VEL3DL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:16]: Looks good and WFPs appear to be in the correct places.

File: GP03FLMA/GP03FLMA_R00006_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00006/instruments/CTDMO_sn37-11704/SBE37*.hex,GP03FLMA-RIM01-02-CTDMOH051,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00006/cg_data/SSIOC_sn13279/STATUS/STA*.DAT,GP03FLMA-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00006/cg_data/SSIOC_sn13279/DOSTA/DOS*.DAT,GP03FLMA-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00006/instruments/PHSEN_sn*/SAMI*.txt,GP03FLMA-RIS01-04-PHSENF000,recovered_inst,Available,
+#mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,#/omc_data/whoi/OMC/GP03FLMA/R00006/cg_data/SSIOC_sn13279/PHSEN/PHS*.DAT,GP03FLMA-RIS01-04-PHSENF000,recovered_host,Not available,No data expected
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00006/cg_data/SSIOC_sn13279/FLORT/FLO*.DAT,GP03FLMA-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GP03FLMA/GP03FLMA_R00006_ingest.csv:34]: Looks good.

File: GP03FLMB/GP03FLMB_R00006_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/instruments/CTDMO_sn37-13448/SBE37*.hex,GP03FLMB-RIM01-02-CTDMOH071,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/cg_data/SSIOC_sn13282/STATUS/STA*.DAT,GP03FLMB-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/cg_data/SSIOC_sn13282/DOSTA/DOS*.DAT,GP03FLMB-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/instruments/PHSEN_snP0079/SAMI*.txt,GP03FLMB-RIS01-04-PHSENF000,recovered_inst,Available,
+#mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/cg_data/SSIOC_sn13282/PHSEN/PHS*.DAT,GP03FLMB-RIS01-04-PHSENF000,recovered_host,Not available,Data not expected
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00006/cg_data/SSIOC_sn13282/FLORT/FLO*.DAT,GP03FLMB-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GP03FLMB/GP03FLMB_R00006_ingest.csv:34]: Looks good.

**Comment by snwhite** [comment]
[Fix GP02HYPM R00006 WFP data](https://trello.com/c/zyNUXzfV/86-fix-gp02hypm-wfp-data)

File: GP02HYPM/GP02HYPM_R00006_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:16]: Looks good.

File: GP03FLMA/GP03FLMA_R00006_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [inline GP03FLMA/GP03FLMA_R00006_ingest.csv:34]: Looks good

File: GP03FLMB/GP03FLMB_R00006_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [inline GP03FLMB/GP03FLMB_R00006_ingest.csv:34]: Looks good

**Pull request merged**